changelog.txt

```diff
@@ -33,6 +33,9 @@ OPENSEADRAGON CHANGELOG
 * Viewer: added a canvas-contextmenu event (#1872 @msalsbery)
 * Added additional documentation for the zoomPerSecond viewer option (#1872 @msalsbery)
 * MouseTracker: Per #1863, dropped support for Internet Explorer < 11 (#1872 @msalsbery)
+* Dropped support for older browsers (IE < 11) (#1949 @msalsbery)
```

Comment:
I think this and the item below should be listed as breaking changes (they should be prefixed by "BREAKING CHANGE" and put at the top above the deprecation). They are not breaking changes the we expect to affect anyone, but it's probably still worth calling them out.

Reply:
> I think this and the item below should be listed as breaking changes (they should be prefixed by "BREAKING CHANGE" and put at the top above the deprecation). They are not breaking changes the we expect to affect anyone, but it's probably still worth calling them out.

Done